Выполнил ДЗ

GSKY90 opened the pull request:
Разметил главную страницу, раскидал классы. Перекинул футер на остальные страницы.

GSKY90 committed:
Выполнил ДЗ

Разметил главную страницу, раскидал классы. Перекинул футер на остальные страницы.

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DLK</title>
+    <style>
+        li {
+         list-style-type: none; 
+        }
+        ul {
+         margin-left: 0; 
+         padding-left: 0; 
+        }
+    </style>
```
Стилизацию внутри html не пишем, я показывал на лекции как нужно, создать отдельную папку и в ней файлы к страницам

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+              <li class="header__title-list"><a href="./catalog/catalog.html">CATALOG</a></li>
+              <li class="header__title-list"><a href="./contact/contact.html">ABOUT US</a></li>
+            </ul>
+            <div>
```
Зачем див если можно сделать списком

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+    <main>
+        <section class="banner">
+            <div>
+                <h1 class="banner__number">01</h1>
```
01 это не заголовок, просто текст

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+                </ul>
+                </div>
+                <h2 class="banner__title">THE OOH NOO COLLECTION</h2>
+                    <p>Designet in Slovenia</p>
```
Прописывай везде классы по бему

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+                    <p>Designet in Slovenia</p>
+                        <button class="banner__title-button" type="button">EXPLORE NOW</button>
+                        <img src="./img/monster.png" alt="main monster" width="635" height="461">
+                    <p><a class="banner__title-info" href="#">Free (continental) USA shipping on all orders of $100 USD or more!</a></p>
```
не нужно оборачивать ссылку в параграф, оставь тут только параграф

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+                    <li><a class="products__title-list" href=""><img src="./img/toys.png" alt="toys" width="280" height="270"><p>TOYS</p></a></li>
+                    <li><a class="products__title-list" href=""><img src="./img/accsessories.png" alt="skate" width="280" height="270"><p>ACCSESSORIES</p></a></li>
+                </ul>
+                    <div>
```
Когда делаешь обертку, классы не забывай прописывать ей

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,170 @@
+            <h5 class="social__title">Connect</h5>
+            <form class="social__form" action="/action_page.php">
+                <label class="social__form" for="fname"></label><br>
+                <input class="social__form" type="text" id="fname" value="Email"><br><br>
```
Зачем 2 <br><br> тут, их вообще не используем. Всё делаем стилизацией

GSKY90 committed:
ДЗ 3

Исправил ошибки с прошлого ДЗ. Разметил СSS главную страницу ( шрифты, фоны, кнопки, картинки )

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,55 @@
+}
+
+body {
+    width: 100%;
+    margin: 0 auto; 
```
Маржин 0 авто не нужно указывать, у тебя будет контейнер где прописано будет свойство

Yan1993-Dev commented on css/style.css:
```diff
@@ -0,0 +1,341 @@
+body {
```
Перенеси в коммон и в стайле не указывай базовую стилизацию

Yan1993-Dev commented on css/style.css:
```diff
@@ -0,0 +1,341 @@
+   font-size: 26px;
+   line-height: 32px;
+   text-transform: uppercase;
+   color: #ffffff;
+   background: #DBE4DB;
+}
+
+.products__title-holiday {
+   font-family: 'Montserrat';
+   font-style: normal;
+   font-weight: 700;
+   font-size: 26px;
+   line-height: 32px;
+   text-transform: uppercase;
+   color: #ffffff;
+   background: #EADDD3;
+}
+   
+   .products__title-toys {
+   font-family: 'Montserrat';
+   font-style: normal;
+   font-weight: 700;
+   font-size: 26px;
+   line-height: 32px;
+   text-transform: uppercase;
+   color: #ffffff;
+   background: #fff3eb;
+}
+      
+
+   .products__title-accsessories {
+   font-family: 'Montserrat';
+   font-style: normal;
+   font-weight: 700;
+   font-size: 26px;
+   line-height: 32px;
+   text-transform: uppercase;
+   color: #ffffff;
+   background: #D5D4E1;
+}
```
Ты можешь сократить запись в виде 
.products__title-apparel,
.products__title-holiday,
.products__title-toys

А вообще укажи им один класс который будешь раскладывать

Yan1993-Dev commented on css/style.css:
```diff
@@ -0,0 +1,341 @@
+}
+
+.products__button-left {
+   border: none;
```
Укажи в коммон для тега button border: none;

GSKY90 committed:
ДЗ

Где я косячник?:)

Yan1993-Dev commented on css/blog.css:
```diff
@@ -0,0 +1,144 @@
+.blog__title {
+    font-family: 'Montserrat';
```
Указывай стиль начертания шрифта sans-serif

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,314 @@
+    header {
+    max-width: 1180px;
+    margin:  0 auto;
+    }
```
У тебя ведь есть контейнер, зачем эта запись тогда?

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,314 @@
+
+    .container {
+        width: 1400px;
+        height: 555px;
```
Высоту убери, она не нужна и паддинг тоже

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,314 @@
+        padding-bottom: 0px;
+    }
+
+    body {
+    background: #FFFFFF;
+    
+    }
```
Перенеси на верх, сначала идет стилизация по тегам, потом по классам

Yan1993-Dev commented on css/common.css:
```diff
@@ -0,0 +1,314 @@
+    
+    }
+
+    main {
+    width: 1180px;
+    margin: 0 auto;
+    }
```
Есть контейнер, тоже самое, убери
